contrib/kubespray/quick-start/services-configuration.yaml.template

```diff
@@ -228,3 +228,18 @@ authentication:
 #  workgroup: WORKGROUP
 #  smbuser: smbuser
 #  smbpwd: smbpwd
+
+device-plugin:
+  service_type: "k8s"
+  {% if "nvidia_device_plugin_yaml_url" in env["cfg"] %}
+  nvidia_device_plugin_yaml_url: "{{ env['cfg']['nvidia_device_plugin_yaml_url'] }}"
+  {% endif %}
+  {% if "amd_device_plugin_yaml_url" in env["cfg"] %}
+  amd_device_plugin_yaml_url: "{{ env['cfg']['amd_device_plugin_yaml_url'] }}"
+  {% endif %}
+  {% if "rdma_config_yaml_url" in env["cfg"] %}
+  rdma_config_yaml_url: "{{ env['cfg']['rdma_config_yaml_url'] }}"
+  {% endif %}
+  {% if "rdma_device_plugin_yaml_url" in env["cfg"] %}
+  rdma_device_plugin_yaml_url: "{{ env['cfg']['rdma_device_plugin_yaml_url'] }}"
+  {% endif %}
```

Comment:
these lines will never be used in this template

Reply:
If you configure `nvidia_device_plugin_yaml_url` in config.yml, they will be used